src/status_im/multiaccounts/login/core.cljs

```diff
@@ -221,26 +289,24 @@
 (defn- keycard-setup? [cofx]
   (boolean (get-in cofx [:db :hardwallet :flow])))
 
-(fx/defn multiaccount-login-success
+(fx/defn multiaccount-login-success-continue
   [{:keys [db] :as cofx}]
-  (let [{:keys [address password save-password? name photo-path creating?]} (:multiaccounts/login db)
-        step (get-in db [:intro-wizard :step])
+  (let [{:keys [address password save-password? creating?]} (:multiaccounts/login db)
         recovering? (recovering-multiaccount? cofx)
         login-only? (not (or creating?
                              recovering?
                              (keycard-setup? cofx)))
         nodes nil]
     (fx/merge cofx
               {:db (-> db
-                       (dissoc :multiaccounts/login)
+                       (assoc :multiaccounts/login {:save-password? save-password?})
```

Comment:
```suggestion
                       ;; :multiaccount/login MUST be dissoc as this is how the app knows the login is successful
                       (dissoc :multiaccounts/login)
```

Reply:
why? `:multiaccounts/login` isn't used for  login is successful decisions

Reply:
it is that is why the `+` menu is spinning forever in this PR